docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -723,13 +708,13 @@ Validation and Verification Criteria
 
 .. math::
 
-  \frac{prevalence_\text{MAM|covered}}{prevalence_\text{MAM|uncovered}} ~ \frac{duration_\text{treated MAM} * E_\text{MAM} + duration_\text{untreated MAM} * (1 - E_{MAM})}{duration_\text{untreated MAM}}
+  \frac{prevalence_\text{MAM|covered}}{prevalence_\text{MAM|uncovered}} ~ \frac{\text{time to response}_\text{effectively treated MAM} * E_\text{MAM} + \text{time to response}_\text{untreated MAM} * (1 - E_{MAM})}{\text{time to response}_\text{untreated MAM}}
```

Comment:
In the previous equations you were using "time to recovery" instead of "time to response". Are these two things supposed to be the same?

Reply:
Yes, those are meant to be the same. Updating now, thanks for catching